requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
+git+https://github.com/edx/XBlock.git#egg=XBlock
```

Comment:
There doesn't seem to be a standard convention that I can see. I think it would be great if we started creating git tags for releases of the XBlock package and pinning XBlock implementation repos to those tags. I'm also happy to defer to any other standard either existing or new.